src/librustc_feature/active.rs

```diff
@@ -565,6 +565,9 @@ declare_features! (
     /// Allow conditional compilation depending on rust version
     (active, cfg_version, "1.45.0", Some(64796), None),
 
+    /// Allows constants of reference type to have integers for the address.
+    (active, const_int_ref, "1.45.0", Some(63197), None),
```

Comment:
Shouldn't there be a dedicated tracking issue? Not sure what the usual process is.

Reply:
We can change that issue to a tracking issue

Reply:
Reading that issue, it contains so much unrelated discussion, I'd prefer a new clean tracking issue.